decision-records/005-repository-organisation-for-v5.md

```diff
@@ -0,0 +1,120 @@
+# Decision Record Title
+
+Update our repository organisation to prepare v5
+<!-- The title should be a short present tense imperative phrase, less than 50
+     characters, like a git commit message. -->
+
+## Decision
+
+<!-- An overview of the decision made. -->
+
+To prepare our work for v5, we've agreed to make the following changes to the files in our repository and their location:
+
+1. Move the sources (currently `src` ) and built package files (currently `package`) under the same npm package. We'll keep shipping only the built files to npm for now, excluding the source files from the package (either via `.npmignore` or using the `files` field of `package.json`)
+2. Remove the built package files (currently `package`) from the versioning
+3. Update our release process to make sure we keep checking the changes (possibly manually) between the package to be released and the previous version
+4. Leave `dist` in its current location
+
+This will yield the following structure for our repository:
+
+```
+govuk-frontend // The root of our repository
+├── ... // Irrelevant files of folder omitted for brevity
+├── dist // Not touching the location, naming might change for clarity
+├── package // Naming might change for clarity
+│   ├── src // Sources are now here, versioned, not shipped to `npm`
+│   │   └── govuk // Structure inside `src` is untouched
+│   ├── dist // Built files are now here, not versioned, ships to npm
```

Comment:
I think the use of 'versioned' here could be confusing (could relate to the version number of the package) so could we use another term?

```suggestion
│   ├── src // Sources are now here, tracked in version control, not shipped to `npm`
│   │   └── govuk // Structure inside `src` is untouched
│   ├── dist // Built files are now here, excluded from version control, ships to npm
```